packages/fuchsia_remote_debug_protocol/lib/src/fuchsia_remote_connection.dart

```diff
@@ -506,6 +507,30 @@ class FuchsiaRemoteConnection {
     _pollDartVms = true;
   }
 
+  /// Helper for getDeviceServicePorts() to extract the vm_service_port from
+  /// json response.
+  List<int> getVmServicePortFromInspectSnapshot(dynamic inspectSnapshot) {
+    final ports = <int>[];
+    for (final item in inspectSnapshot as List<dynamic>) {
+      if (item['payload'] == null || !(item as Map).containsKey('payload'))
```

Comment:
this needs curly braces around the block, per `curly_braces_in_flow_control_structures`